### DISCUSSION TITLE
How to replace import {     DrawerContentScrollView,     DrawerItem } from '@react-navigation/drawer'  in expo/router

### DISCUSSION THREAD

**Discussion by raulgonzalezdev** [expo/router#55] [Q&A] [answered]
Hi, community, I love the work, you are doing with expo/router, I have a question how to replace these imports using expo/router,

`import {     DrawerContentScrollView,     DrawerItem } from '@react-navigation/drawer'`

 in the migration tutorial they only mention these

Navigators

createStackNavigator ====> Stack
createBottomTabNavigator ====>Tabs
createDrawerNavigator ====>Drawer
createNativeStackNavigator ====>NativeStack

**Comment by trin4ik** [comment]
as i see,  `DrawerContentScrollView` is just [wrapper with insets for ScrollView](https://github.com/react-navigation/react-navigation/blob/50b88d40496a04f613073c63119b21a104ec9bc2/packages/drawer/src/views/DrawerContentScrollView.tsx), in other words sugar.
i think at now you can use 
`import { DrawerContentScrollView, DrawerItem } from '@react-navigation/drawer'`
or write your own wrappers

**Comment by EvanBacon** [comment, marked as answer]
@gqcryptoraul we are still trying to determine how we want to re-export the main navigators. Importing directly from `@react-navigation/drawer` is the recommended solution for now.

**Reply by hugobayoud** [reply]
@EvanBacon 
Hey!
Does this recommended solution you gave still the good one now? I still can't find any `DrawerContentScrollView` in the expo-router v3.
